### ISSUE TITLE
Is there an option to disable slugified URLs?

### ISSUE THREAD

**Issue by sjames272727** [transitive-bullshit/nextjs-notion-starter-kit#121]
I have a notion with multiple pages of the same title, so slugified URLs break. I don't mind having a URL with the full ID, is there any way to disable slugifying URLs?

**Comment by sjames272727** [comment]
After a bit of detective work I figured it out.

I realized that the URLs are not slugified in production, so there must've been a toggle somewhere.

This seemed curious https://github.com/transitive-bullshit/nextjs-notion-starter-kit/blob/main/lib/map-page-url.ts#L5

So I set `includeNotionIdInUrls ` to true in the site config, every URL is not slugified now!

**Comment by sjames272727** [comment]
You should really consider pinning this issue, or at least adding the `includeNotionIdInUrls` config option to `site.config.js` template.

Anyways, thank you for this amazing project, my site looks amazing!

**Comment by MHC-I** [comment]
I have the same problem as you. Can you tell me in detail how you solved it？

**Comment by transitive-bullshit** [comment]
@MHC-I add `"includeNotionIdInUrls": false` as part of the JSON in your `site.config.js`.

**Comment by MHC-I** [comment]
@transitive-bullshit  Yeah, I failed to reload just now. Now I have solved it. Thank you very much. This is a good project.
